blueocean-pipeline-api-impl/src/main/java/io/jenkins/blueocean/rest/impl/pipeline/PipelineNodeGraphVisitor.java

```diff
@@ -274,9 +274,13 @@ public void parallelStart(@Nonnull FlowNode parallelStartNode, @Nonnull FlowNode
                     status = new NodeRunStatus(genericStatus);
                 }
             }else{
-                times = new TimingInfo(TimingAction.getStartTime(branchStartNode)+System.currentTimeMillis(),
+                long startTime = System.currentTimeMillis();
```

Comment:
See the JIRA - the `TimingAction` is only missing for a very brief window between creation of the `FlowNode` and attaching the `TimingAction`, always, so far as I can tell. @svanoort - does that sound right?